DirectEmployers/MyJobs
# [PD 2635] new partner set primary contact

## What's this PR do?
This PR will set the primary contact as the first contact entered when a new partner is created inside of NUO

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of NUO and create a new partner with a new contact or multiple contacts to go with it

## How should this be manually tested?
Pull down the repository. Go to NUO app and create a new partner. You will have to create at least 1 contact or you can create multiples. Finish the application and submit the form to create the new parter. Afterwards to test, go into PRM and search for the partner you created. You should see the card shows the first contact or ONLY contact that was created with the partner. You can also go into edit that partner and see the primary contact has been set inside of the contacts dropdown list as well.

## Any background context you want to provide?
Not too sure if test need to be made for this but if so I would need assistance as far as that goes for writing the test. Another Python writing test for myself lol.
